taskt/Core/EncryptionServices.cs

```diff
@@ -1,27 +1,24 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Linq;
 using System.Security.Cryptography;
 using System.Text;
-using System.Threading.Tasks;
 
 namespace taskt.Core
 {
     public static class EncryptionServices
     {
         // This size of the IV (in bytes) must = (keysize / 8).  Default keysize is 256, so the IV must be
         // 32 bytes long.  Using a 16 character string here gives us 32 bytes when converted to a byte array.
-        private const string initVector = "kPcmAhmtL4ofXSMJ";
+        private const string InitVector = "kPcmAhmtL4ofXSMJ";
         // This constant is used to determine the keysize of the encryption algorithm
-        private const int keysize = 256;
+        private const int KeySize = 256;
```

Comment:
The fields are private can you change to `_initVector` and `_keySize`?

Reply:
Do you remember, we agreed upon the **(PascalCase)** naming convention for **Contants** and Enums?

Reply:
Did we? I don't remember that

Reply:
Yes, we did.

Reply:
This still needs to be adjusted

Reply:
updated